1.7/administration/installing/cloud/aws/basic/index.md

```diff
@@ -0,0 +1,92 @@
+---
+post_title: Basic DC/OS Installation on AWS
```

Comment:
The option is awesome. I'd love for there to be a "if you'd like to customize more go here" at the top. It feels weird, as a user, to be presented with a list of options instead of the content immediately.